performance_tests/mnist_10k_test.amlg

```diff
@@ -100,9 +100,8 @@
 	(assign (assoc
 		k (get params (list "hyperparameter_map" "k"))
 		p (get params (list "hyperparameter_map" "p"))
-		deviations (get params (list "hyperparameter_map" "useDeviations"))
 	))
-	(print "MNIST 10k params (k p deviations): " k " " p " " deviations "\n- - - - -\n")
+	(print "MNIST 10k params (k p): " k " " p "\n- - - - -\n")
```

Comment:
I think there's some unused `use_devations` stuff at the top of this file. Could you clean that up as well? :)

Reply:
If `use_deviations` is still gonna be an analyze parameter, just add it to the analyze call please.